glossary/en/terms/internet/index.md

```diff
@@ -4,4 +4,4 @@ lang: en
 title: Internet
 ---
 
```

Comment:
@jgkim -This is good! I just notice that we have the same definition to Internet and Information (which is wrong). I'll fix it after I'll merge this pull request.